src/main/java/ch/heigvd/statique/utils/Server.java

```diff
@@ -0,0 +1,97 @@
+package ch.heigvd.statique.utils;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import org.apache.commons.io.FileUtils;
+
+import java.io.*;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.logging.Logger;
+
+public class Server {
+    private final static Logger LOG = Logger.getLogger(Server.class.getName());
+    private final Path site;
+    private final int port;
+
+    /**
+     * Server constructor
+     *
+     * @param site path to site
+     * @param port server port number
+     */
+    public Server(Path site, int port) {
+        this.site = site;
```

Comment:
This is for handle the 403 error correctly, comment on the line 63

```suggestion
        this.site = site.toAbsolutePath().normalize();
```